### DISCUSSION TITLE
sMite It is not compatible with Xiaomi, help me please

### DISCUSSION THREAD

**Discussion by KunoichiUra** [thatskymod/Sky-CotL-Scripts#41] [Q&A]
The Canvas app starts blinking, I uninstalled the Canvas sMite and then it works ...However on the Samsung it works great! I need smite for Xiaomi,please,how can I do? thank you

**Comment by Sagimoster** [comment]
Tha same problem me to

**Comment by SamuelGrey** [comment]
I imported sMite into canvas on my xiaomi phone,however when i run sky and mark sMite it doesnt show up at all...If anyone knows how to fix this please help us 🥺

**Comment by thatskymod** [comment]
As of today, the latest version of libsmite (1.2.0) is not compatible with any phone brand and operating system, not just Xiaomi. The only solution for now is to avoid libsmite and opt for alternatives. For instance, [libLangit033.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/93c2b368edf146cfdde8d50c1bb6ff818fdaaba4/Canvas/Libs/libLangit033.so) and [libRomanBeta.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/516c267426b36f1cf0a8b1bab1acf7568c712a92/Canvas/Libs/libRomanBeta.so) are viable alternatives to mitigate the global outage of libsmite.
